From 5e81d8bd693bd05ba09ce6d75c66ac06830f643e Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Tue, 6 Dec 2022 11:50:06 +0200
Subject: [PATCH 29/54] s32g: fdts: Add SCMI GPIO node

Issue: ALB-9555
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32g.dtsi | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/fdts/s32g.dtsi b/fdts/s32g.dtsi
index da4a45ad2..31bf3d300 100644
--- a/fdts/s32g.dtsi
+++ b/fdts/s32g.dtsi
@@ -31,6 +31,22 @@
 		};
 	};
 
+	firmware {
+		scmi {
+			gpio_scmi: protocol@81 {
+				reg = <0x81>;
+				#gpio-cells = <2>;
+				gpio-controller;
+				#interrupt-cells = <2>;
+				interrupt-controller;
+
+				/* GPIO ranges : 0-101 & 112-190 */
+				gpio-ranges = <&pinctrl 0   0   102>,
+					      <&pinctrl 112 112 79>;
+			};
+		};
+	};
+
 	clocks {
 		pfe_mac0_rmii: pfe_mac0_rmii {
 			compatible = "fixed-clock";
-- 
2.25.1

