From 6ea5a376e3661f04cce5c1f4fd751d250e759ccb Mon Sep 17 00:00:00 2001
From: Radu Pirea <radu-nicolae.pirea@nxp.com>
Date: Tue, 19 Jul 2022 17:28:16 +0300
Subject: [PATCH 009/102] i2c: mxc: sort include files

Issue: ALB-9034

Upstream-Status: Pending 

Signed-off-by: Radu Pirea <radu-nicolae.pirea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/i2c/mxc_i2c.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/i2c/mxc_i2c.c b/drivers/i2c/mxc_i2c.c
index a81e2fd951..8b572dc7ce 100644
--- a/drivers/i2c/mxc_i2c.c
+++ b/drivers/i2c/mxc_i2c.c
@@ -19,14 +19,14 @@
 #if defined(CONFIG_MACH_IMX)
 #include <asm/arch/imx-regs.h>
 #endif
-#include <dm/device_compat.h>
-#include <linux/errno.h>
-#include <asm/mach-imx/mxc_i2c.h>
-#include <asm/io.h>
+#include <dm.h>
 #include <i2c.h>
 #include <watchdog.h>
-#include <dm.h>
+#include <asm/io.h>
+#include <asm/mach-imx/mxc_i2c.h>
+#include <dm/device_compat.h>
 #include <dm/pinctrl.h>
+#include <linux/errno.h>
 
 #define I2C_QUIRK_FLAG		(1 << 0)
 
-- 
2.17.1

