From b4d11322a0d49a8e65b18fc3ce1a9d3c5036a8ff Mon Sep 17 00:00:00 2001
From: Bogdan Roman <bogdan-gabriel.roman@nxp.com>
Date: Wed, 15 Feb 2023 17:21:41 +0200
Subject: [PATCH 2/9] pkcs11-hse: add examples target in top directory Makefile

Issue: ALB-9734
Upstream-Status: Pending 

Signed-off-by: Bogdan Roman <bogdan-gabriel.roman@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 Makefile | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index da5d663..ebe5fa4 100644
--- a/Makefile
+++ b/Makefile
@@ -62,7 +62,7 @@ INCL = -I$(HSE_FWDIR)/interface                                                \
 
 DEFS := -DUIO_DEV=$(UIO_DEV)
 
-all: $(PKCS_LIB)
+all: $(PKCS_LIB) examples
 
 $(PKCS_LIB): $(HSE_LIB).$(HSE_LIBVER) $(PKCS_OBJS)
 	$(CC) -shared $(CFLAGS) -L$(shell pwd) $(LDFLAGS) $(PKCS_OBJS) -o $@ -lhse
@@ -83,8 +83,13 @@ $(HSE_ODIR)/%.o: $(HSE_SDIR)/%.c $(HSE_ODIR)
 $(HSE_ODIR):
 	mkdir -p $@
 
+.PHONY: examples
+examples:
+	make -C examples PKCS11HSE_DIR=$(CURDIR)
+
 clean:
 	rm -f *.so*
 	rm -rf $(PKCS_ODIR) $(HSE_ODIR)
+	make -C examples clean
 
 .PHONY: clean all
-- 
2.25.1

