From d19fba8bb3c3e667271582cfe860eda0a37e9127 Mon Sep 17 00:00:00 2001
From: Zhantao Tang <zhantao.tang@windriver.com>
Date: Sun, 18 Feb 2024 10:33:22 +0000
Subject: [PATCH] s32: env: update fit_config_header to compatible with
 oe-core

Upstream-Status: Inappropriate [WR Linux specific]

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32-cc.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/s32-cc.h b/include/configs/s32-cc.h
index 49c991b92f..2758ccd8f0 100644
--- a/include/configs/s32-cc.h
+++ b/include/configs/s32-cc.h
@@ -149,7 +149,7 @@
 	"image=Image\0" \
 	"fitimage=fitImage\0" \
 	"fitimage_addr=" __stringify(S32CC_FIT_ADDR) "\0" \
-	"fit_config_header=conf-\0" \
+	"fit_config_header=conf-freescale_\0" \
 	"initrd_high=" __stringify(S32CC_INITRD_HIGH_ADDR) "\0" \
 	"ipaddr=" S32CC_IPADDR "\0"\
 	"loadfdt=fatload mmc ${mmcdev}:${mmcpart} ${fdt_addr} ${fdt_file}; " \
-- 
2.35.5

