From ffd3e320674389fae270c69a3b560c19d66e3f49 Mon Sep 17 00:00:00 2001
From: Ionut Vicovan <Ionut.Vicovan@nxp.com>
Date: Wed, 2 Nov 2022 23:33:30 +0200
Subject: [PATCH 07/19] s32cc: pcie: Check if the pcie node is enabled

Issue: ALB-9211

Upstream-Status: Pending 

Signed-off-by: Ionut Vicovan <Ionut.Vicovan@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/pci/pcie_s32gen1.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/pci/pcie_s32gen1.c b/drivers/pci/pcie_s32gen1.c
index 7a0e417969..5e5e8598cf 100644
--- a/drivers/pci/pcie_s32gen1.c
+++ b/drivers/pci/pcie_s32gen1.c
@@ -821,6 +821,13 @@ static int s32_pcie_get_config_from_device_tree(struct s32_pcie *pcie)
 
 	debug("%s: dt node: %s\n", __func__, dev_read_name(dev));
 
+	/* check if node is enabled */
+	if (!dev_read_enabled(dev)) {
+		debug("%s: %s: Node is disabled\n", __func__,
+		      dev_read_name(dev));
+		return -ENODEV;
+	}
+
 	/* check if serdes is enabled for the pcie node */
 	ret = s32gen1_check_serdes(dev);
 	if (ret)
-- 
2.17.1

