From 331e8881871b69324b1a53539a1dff7b2715939b Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 13 May 2022 13:52:45 +0300
Subject: [PATCH 08/50] fdts: s32cc: Make FXOSC part of the soc node

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 20 +++++++++++++-------
 1 file changed, 13 insertions(+), 7 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index a5f2ed002..6581bb7a3 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -1,7 +1,6 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
 /*
  * Copyright 2020-2022 NXP
- *
- * SPDX-License-Identifier: BSD-3-Clause
  */
 
 #include <dt-bindings/clock/s32gen1-clock-freq.h>
@@ -23,11 +22,18 @@
 		ranges;
 	};
 
-	fxosc: fxosc@40050000 {
-		compatible = "fixed-clock", "nxp,s32cc-fxosc";
-		reg = <0x0 0x40050000 0x0 0x3000>;
-		clock-frequency = <S32GEN1_FXOSC_FREQ>;
-		#clock-cells = <1>;
+	soc {
+		compatible = "simple-bus";
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges = <0x0 0x40000000 0x0 0x40000000 0x0 0x14000000>;
+
+		fxosc: fxosc@40050000 {
+			compatible = "fixed-clock", "nxp,s32cc-fxosc";
+			reg = <0x0 0x40050000 0x0 0x3000>;
+			clock-frequency = <S32GEN1_FXOSC_FREQ>;
+			#clock-cells = <1>;
+		};
 	};
 
 	firc: firc {
-- 
2.17.1

