From 05e91589ec913b0eddc35d82d46163b8a4aaff3b Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 13 May 2022 14:37:06 +0300
Subject: [PATCH 15/50] fdts: s32cc: Make ARM PLL part of the soc node

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index f0ceb61de..6d4f7df26 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -28,6 +28,18 @@
 		#size-cells = <2>;
 		ranges = <0x0 0x40000000 0x0 0x40000000 0x0 0x14000000>;
 
+		armpll: armpll@40038000 {
+			compatible = "nxp,s32cc-armpll";
+			reg = <0x0 0x40038000 0x0 0x3000>;
+
+			assigned-clocks =
+				<&plat_clks S32GEN1_CLK_ARM_PLL_MUX>,
+				<&plat_clks S32GEN1_CLK_ARM_PLL_VCO>,
+				<&plat_clks S32GEN1_CLK_ARM_PLL_PHI0>;
+			assigned-clock-parents =
+				<&plat_clks S32GEN1_CLK_FXOSC>;
+		};
+
 		fxosc: fxosc@40050000 {
 			compatible = "fixed-clock", "nxp,s32cc-fxosc";
 			reg = <0x0 0x40050000 0x0 0x3000>;
@@ -192,18 +204,6 @@
 			<0>, <0>, <0>, <0>,
 			<0>, <0>, <100000000>;
 
-		armpll: armpll@4003800 {
-			compatible = "nxp,s32cc-armpll";
-			reg = <0x0 0x40038000 0x0 0x3000>;
-
-			assigned-clocks =
-				<&plat_clks S32GEN1_CLK_ARM_PLL_MUX>,
-				<&plat_clks S32GEN1_CLK_ARM_PLL_VCO>,
-				<&plat_clks S32GEN1_CLK_ARM_PLL_PHI0>;
-			assigned-clock-parents =
-				<&plat_clks S32GEN1_CLK_FXOSC>;
-		};
-
 		armdfs: armdfs@40054000 {
 			compatible = "nxp,s32cc-armdfs";
 			reg = <0x0 0x40054000 0x0 0x3000>;
-- 
2.17.1

