From afeba432a7e61e51191c5b829d41b5d42d16cb4a Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 13 May 2022 14:43:14 +0300
Subject: [PATCH 18/50] fdts: s32cc: Make ARM DFS part of the soc node

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index 05ef30103..8913c8ecd 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -83,6 +83,18 @@
 				<800000000>;
 		};
 
+		periphdfs: periphdfs@40058000 {
+			compatible = "nxp,s32cc-periphdfs";
+			reg = <0x0 0x40058000 0x0 0x3000>;
+
+			assigned-clocks =
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_DFS1>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_DFS3>;
+			assigned-clock-rates =
+				<S32GEN1_PERIPH_DFS1_FREQ>,
+				<S32GEN1_PERIPH_DFS3_FREQ>;
+		};
+
 		wkpu: wkpu@40090000 {
 			compatible = "nxp,s32cc-wkpu";
 			reg = <0x0 0x40090000 0x0 0x10000>, /* WKPU */
@@ -240,18 +252,6 @@
 			<0>, <0>, <0>, <0>,
 			<0>, <0>, <100000000>;
 
-		periphdfs: armdfs@40058000 {
-			compatible = "nxp,s32cc-periphdfs";
-			reg = <0x0 0x40058000 0x0 0x3000>;
-
-			assigned-clocks =
-				<&plat_clks S32GEN1_CLK_PERIPH_PLL_DFS1>,
-				<&plat_clks S32GEN1_CLK_PERIPH_PLL_DFS3>;
-			assigned-clock-rates =
-				<S32GEN1_PERIPH_DFS1_FREQ>,
-				<S32GEN1_PERIPH_DFS3_FREQ>;
-		};
-
 		accelpll: accelpll@40040000 {
 			compatible = "nxp,s32cc-accelpll";
 			reg = <0x0 0x40040000 0x0 0x3000>;
-- 
2.17.1

