From 001f0f958effdc16d36220073e669cabc8fa3ae0 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 13 May 2022 14:49:09 +0300
Subject: [PATCH 20/50] fdts: s32cc: Make DDR PLL part of the soc node

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index 6fa21063f..b9130f22b 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -69,6 +69,11 @@
 			reg = <0x0 0x40040000 0x0 0x3000>;
 		};
 
+		ddrpll: ddrpll@40044000 {
+			compatible = "nxp,s32cc-ddrpll";
+			reg = <0x0 0x40044000 0x0 0x3000>;
+		};
+
 		fxosc: fxosc@40050000 {
 			compatible = "fixed-clock", "nxp,s32cc-fxosc";
 			reg = <0x0 0x40050000 0x0 0x3000>;
@@ -257,11 +262,6 @@
 			<0>, <0>, <0>, <0>,
 			<0>, <0>, <100000000>;
 
-		ddrpll: ddrpll@40044000 {
-			compatible = "nxp,s32cc-ddrpll";
-			reg = <0x0 0x40044000 0x0 0x3000>;
-		};
-
 		mc_me: mc_me@40088000 {
 			compatible = "nxp,s32cc-mc_me";
 			reg = <0x0 0x40088000 0x0 0x3000>;
-- 
2.17.1

