From 01f1c3103cda3a5fe194997780f748d935e7e571 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 13 May 2022 15:00:13 +0300
Subject: [PATCH 25/50] fdts: s32cc: Make MC_CGM1 part of th soc node

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index 133aa43e1..32eff6483 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -90,6 +90,18 @@
 				<S32GEN1_QSPI_2X_CLK_FREQ>;
 		};
 
+		mc_cgm1: mc_cgm1@40034000 {
+			compatible = "nxp,s32cc-mc_cgm1";
+			reg = <0x0 0x40034000 0x0 0x3000>;
+
+			assigned-clocks =
+				<&plat_clks S32GEN1_CLK_MC_CGM1_MUX0>,
+				<&plat_clks S32GEN1_CLK_A53_CORE>;
+			assigned-clock-parents =
+				<&plat_clks S32GEN1_CLK_ARM_PLL_PHI0>,
+				<0>;
+		};
+
 		armpll: armpll@40038000 {
 			compatible = "nxp,s32cc-armpll";
 			reg = <0x0 0x40038000 0x0 0x3000>;
@@ -339,18 +351,6 @@
 			<0>, <0>, <0>, <0>,
 			<0>, <0>, <100000000>;
 
-		mc_cgm1: mc_cgm1@40034000 {
-			compatible = "nxp,s32cc-mc_cgm1";
-			reg = <0x0 0x40034000 0x0 0x3000>;
-
-			assigned-clocks =
-				<&plat_clks S32GEN1_CLK_MC_CGM1_MUX0>,
-				<&plat_clks S32GEN1_CLK_A53_CORE>;
-			assigned-clock-parents =
-				<&plat_clks S32GEN1_CLK_ARM_PLL_PHI0>,
-				<0>;
-		};
-
 		mc_cgm5: mc_cgm5@40068000 {
 			compatible = "nxp,s32cc-mc_cgm5";
 			reg = <0x0 0x40068000 0x0 0x3000>;
-- 
2.17.1

