From acdb3d1a6833fad6cef265f36a15f5871e36c435 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 13 May 2022 15:01:37 +0300
Subject: [PATCH 26/50] fdts: s32cc: Add MC_CGM5 to the soc node

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index 32eff6483..ef81a08f6 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -179,6 +179,21 @@
 				<S32GEN1_PERIPH_DFS3_FREQ>;
 		};
 
+		mc_cgm5: mc_cgm5@40068000 {
+			compatible = "nxp,s32cc-mc_cgm5";
+			reg = <0x0 0x40068000 0x0 0x3000>;
+
+			assigned-clocks =
+				<&plat_clks S32GEN1_CLK_MC_CGM5_MUX0>,
+				<&plat_clks S32GEN1_CLK_DDR>;
+			assigned-clock-parents =
+				<&plat_clks S32GEN1_CLK_DDR_PLL_PHI0>,
+				<0>;
+			assigned-clock-rates =
+				<0>,
+				<S32GEN1_DDR_FREQ>;
+		};
+
 		rgm: rgm@40078000 {
 			compatible = "nxp,s32cc-rgm";
 			reg = <0x0 0x40078000 0x0 0x3000>;
@@ -350,21 +365,6 @@
 		assigned-clock-rates =
 			<0>, <0>, <0>, <0>,
 			<0>, <0>, <100000000>;
-
-		mc_cgm5: mc_cgm5@40068000 {
-			compatible = "nxp,s32cc-mc_cgm5";
-			reg = <0x0 0x40068000 0x0 0x3000>;
-
-			assigned-clocks =
-				<&plat_clks S32GEN1_CLK_MC_CGM5_MUX0>,
-				<&plat_clks S32GEN1_CLK_DDR>;
-			assigned-clock-parents =
-				<&plat_clks S32GEN1_CLK_DDR_PLL_PHI0>,
-				<0>;
-			assigned-clock-rates =
-				<0>,
-				<S32GEN1_DDR_FREQ>;
-		};
 	};
 
 	ocotp: ocotp@400a4000 {
-- 
2.17.1

