From d9e21805eac18b355fb1464b071dfbb844ac50de Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 13 May 2022 15:13:22 +0300
Subject: [PATCH 27/50] fdts: s32cc: Make OCOTP part of the soc node

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index ef81a08f6..0912c8ba2 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -220,6 +220,11 @@
 			status = "disabled";
 		};
 
+		ocotp: ocotp@400a4000 {
+			reg = <0x0 0x400a4000 0x0 0x400>;
+			status = "okay";
+		};
+
 		i2c0: i2c@401e4000 {
 			compatible = "nxp,s32cc-i2c";
 			reg = <0x0 0x401e4000 0x0 0x10000>;
@@ -367,11 +372,6 @@
 			<0>, <0>, <100000000>;
 	};
 
-	ocotp: ocotp@400a4000 {
-		reg = <0x0 0x400a4000 0x0 0x400>;
-		status = "okay";
-	};
-
 	qspi: quadspi@40134000 {
 		compatible = "nxp,s32cc-qspi";
 		#address-cells = <1>;
-- 
2.17.1

