From f4c34168abfff261abf0b1b3fb79ffaa29dae607 Mon Sep 17 00:00:00 2001
From: Ciprian Costea <ciprianmarian.costea@nxp.com>
Date: Thu, 26 May 2022 12:27:06 +0300
Subject: [PATCH 47/50] s32g: evb: usb: Add clock

Issue: ALB-8606
Upstream-Status: Pending 

Signed-off-by: Ciprian Costea <ciprianmarian.costea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32gxxxa-evb.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/fdts/s32gxxxa-evb.dtsi b/fdts/s32gxxxa-evb.dtsi
index 56ff7f3c7..0fda509ef 100644
--- a/fdts/s32gxxxa-evb.dtsi
+++ b/fdts/s32gxxxa-evb.dtsi
@@ -13,9 +13,12 @@
 	usbotg: usb@44064000 {
 		compatible = "fsl,imx27-usb";
 		reg = <0x0 0x44064000 0x0 0x200>;
+		clocks = <&clks S32G_SCMI_CLK_USB_MEM>,
+			   <&clks S32G_SCMI_CLK_USB_LOW>;
 		pinctrl-0 = <&pinctrl0_usb &pinctrl1_usb>;
 		pinctrl-names = "default";
 		phys = <&saf1508_phy>;
+		dr_mode = "host";
 	};
 
 	saf1508_phy: usb_phy@44064000 {
-- 
2.17.1

