From 3afc995c0f49b04f0ab5c04aba27eef44a95266a Mon Sep 17 00:00:00 2001
From: Ciprian Costea <ciprianmarian.costea@nxp.com>
Date: Wed, 16 Mar 2022 10:14:27 +0200
Subject: [PATCH 004/103] nvme: rdbX: Enable NVME CMD support

Issue: ALB-8658
Upstream-Status: Pending 

Signed-off-by: Ciprian Costea <ciprianmarian.costea@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 board/freescale/s32-gen1/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/freescale/s32-gen1/Kconfig b/board/freescale/s32-gen1/Kconfig
index 3d2ea3c1c0..8d445d709c 100644
--- a/board/freescale/s32-gen1/Kconfig
+++ b/board/freescale/s32-gen1/Kconfig
@@ -12,6 +12,7 @@ config NXP_S32GRDB_BOARD
 	bool
 	default y
 	depends on TARGET_S32G274ARDB2 || TARGET_S32G399ARDB3
+	imply NVME
 
 config S32GEN1_DRAM_INLINE_ECC
 	bool "S32 GEN1 inline ECC"
-- 
2.25.1

