From 2f989e2d93487a777904fc026c05a2f2eba19b1e Mon Sep 17 00:00:00 2001
From: Bogdan Roman <bogdan-gabriel.roman@nxp.com>
Date: Wed, 6 Apr 2022 15:06:49 +0300
Subject: [PATCH 091/103] s32: i2c: remove unnecessary code from mxc_i2c.c

Issue: ALB-8594
Upstream-Status: Pending 

Signed-off-by: Bogdan Roman <bogdan-gabriel.roman@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/i2c/mxc_i2c.c | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/drivers/i2c/mxc_i2c.c b/drivers/i2c/mxc_i2c.c
index a792690f15..39313928ab 100644
--- a/drivers/i2c/mxc_i2c.c
+++ b/drivers/i2c/mxc_i2c.c
@@ -140,8 +140,6 @@ static u16 i2c_clk_div[50][2] = {
 #define CONFIG_SYS_MXC_I2C5_SLAVE 0
 #endif
 
-int enable_i2c_clk(unsigned char enable, unsigned int i2c_num);
-
 /*
  * Calculate and set proper clock divider
  */
@@ -700,8 +698,7 @@ static int bus_i2c_write(struct mxc_i2c_bus *i2c_bus, u8 chip, u32 addr,
 
 static struct mxc_i2c_bus mxc_i2c_buses[] = {
 #if defined(CONFIG_LS1021A) || defined(CONFIG_VF610) || \
-	defined(CONFIG_FSL_LAYERSCAPE) || \
-	defined(CONFIG_NXP_S32CC)
+	defined(CONFIG_FSL_LAYERSCAPE)
 	{ 0, I2C1_BASE_ADDR, I2C_QUIRK_FLAG },
 	{ 1, I2C2_BASE_ADDR, I2C_QUIRK_FLAG },
 	{ 2, I2C3_BASE_ADDR, I2C_QUIRK_FLAG },
@@ -890,14 +887,6 @@ static int mxc_i2c_set_bus_speed(struct udevice *bus, unsigned int speed)
 	return bus_i2c_set_bus_speed(i2c_bus, speed);
 }
 
-__attribute__((weak))
-int gpio_request_by_name_nodev(ofnode node, const char *list_name, int index,
-			       struct gpio_desc *desc, int flags)
-{
-	// To be overridden
-	return -1;
-}
-
 static int mxc_i2c_probe(struct udevice *bus)
 {
 	struct mxc_i2c_bus *i2c_bus = dev_get_priv(bus);
-- 
2.25.1

